fix: Adding missed network configurations (#13)
ibm-cloud-docs/security-groups: sg_network_config.md

A security group augments any existing network configuration. Therefore, a security group cannot span across networks that cannot communicate with one another.
{: shortdesc}

> Security groups are interface constructs, they're not IP specific. When a virtual server has public interface with security group applied, it is applicable to its assigned(with the interface) portal subnet also.

Reviewer: ```suggestion
Security groups are interface constructs, they're not IP specific. When a virtual server has public interface with security group applied, the security group also applies to the interfaces assigned portable subnet.
```

Author: Noted. Patching it as commit

> In order to prevent DHCP spoofing, a DROP rule is added as a mandate whenever the secuirty group is applied. so, trying to set DHCP server using secondary portable subnet will not work with security applied.

Reviewer: ```suggestion
In order to prevent DHCP spoofing, a DROP rule is added as a mandate whenever the secuirty group is applied. As such, trying to set DHCP server using secondary portable subnet will not work with a security group applied.
```

Author: My bad. patching this too.
